app/services/feeds/assemble_article_markdown.rb

```diff
@@ -31,6 +31,14 @@ def call
 
     private
 
+    def feed_url
+      @feed_url ||=
+        @feed.url.presence ||
```

Comment:
is there a good reason to prefer @feed.url (from links) even if it is present? From what I can tell from https://validator.w3.org/feed/docs/atom.html#recommendedFeedElements ,  that implementation in feedjira is a little optimistic (it just uses the last link that's not the feed itself ?).